### DISCUSSION TITLE
DÚVIDAS SOBRE CONSULTA DE DISTRIBUIÇÃO DF-e

### DISCUSSION THREAD

**Discussion by EduardoFerreira22** [TadaSoftware/PyNFe#346] [General]
Olá,
Minha dúvida é se para usar o [[Consultar Distribuição DF-e](https://github.com/TadaSoftware/PyNFe/wiki/Consulta-Distribui%C3%A7%C3%A3o)]  se os documentos já vem completos ou seria necessário manifestar?

já nessa outra documentação [Fluxo da Consulta de Distribuição por NSU](https://github.com/TadaSoftware/PyNFe/wiki/Fluxo-da-Consulta-de-Distribuicao-por-NSU) apresenta-se como se o xml fosse retornado pela metade até que seja manifestado.

Estou usando o código a cima e descobri que está sendo retornado apenas as notas que já foram manifestadas pelo cliente, as que ainda não foram manifestadas, não estão vindo na requisição.

Anexei um print de como estou utilizando o código atualmente em uma aplicação WEB com Django. 
![buscar_sefaz](https://github.com/TadaSoftware/PyNFe/assets/104332245/aa2e51d5-e0ad-45bf-99fd-0a05341e468f)

Desde já agradeço, e aguardo por ajuda para entender a forma correta de uso.

**Comment by leogregianin** [comment]
@EduardoFerreira22, para receber o XML completo é necessário manifestar.
Sem a manifestação, retorna somente o resumo da nota.

**Reply by EduardoFerreira22** [reply]
Poderia me dar um exemplo de como posso adaptar e usar o código da forma correta?

**Reply by EduardoFerreira22** [reply]
Ou se esse código que te mostrei está sendo usado da forma correta?
